packages/application-generic/src/factories/sms/handlers/kavenegar-novu-sms.handler.ts

```diff
@@ -0,0 +1,16 @@
+import { KavenegarNovuSmsSmsProvider } from '@novu/kavenegar-novu-sms';
+import { ChannelTypeEnum, ICredentials } from '@novu/shared';
+import { BaseSmsHandler } from './base.handler';
+
+export class KavenegarNovuSmsHandler extends BaseSmsHandler {
+  constructor() {
+    super('kavenegar-novu-sms', ChannelTypeEnum.SMS);
+  }
+
+  buildProvider(credentials: ICredentials) {
+    this.provider = new KavenegarNovuSmsSmsProvider({
+      apiKey: credentials.apiKey,
+      from: credentials.from,
+    });
+  }
+}
```

Comment:
```suggestion
import { KavenegarSmsProvider } from '@novu/kavenegar';
import { ChannelTypeEnum, ICredentials } from '@novu/shared';
import { BaseSmsHandler } from './base.handler';

export class KavenegarHandler extends BaseSmsHandler {
  constructor() {
    super('kavenegar', ChannelTypeEnum.SMS);
  }

  buildProvider(credentials: ICredentials) {
    this.provider = new KavenegarSmsProvider({
      apiKey: credentials.apiKey,
      from: credentials.from,
    });
  }
}
```
Because branding.